contracts/middle-layer/resource-mirror/encode/CmnEncode.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.0;
+
+import "../storage/CmnStorage.sol";
+
+contract CmnEncode is CmnStorage {
+    /*----------------- encode -----------------*/
+    function encodeCmnMirrorAckPackage(CmnMirrorAckPackage calldata mirrorAckPkg) external pure returns (bytes memory) {
+        return wrapEncode(TYPE_MIRROR, abi.encode(mirrorAckPkg));
+    }
+
+    function encodeCmnDeleteSynPackage(CmnDeleteSynPackage calldata synPkg) external pure returns (bytes memory) {
+        return wrapEncode(TYPE_DELETE, abi.encode(synPkg));
+    }
+
+    function encodeExtraData(ExtraData calldata _extraData) external pure returns (bytes memory) {
```

Comment:
This function seems useless anymore. abi.encode(_extraData) is enough.

Reply:
fixed